backend/corpora/common/entities/entity.py

```diff
@@ -62,3 +62,30 @@ def __getattr__(self, name):
         :return:
         """
         return self.db_object.__getattribute__(name)
+
+    @classmethod
+    def _create_sub_objects(
+        cls, rows: typing.List[dict], db_table: Base, keys: typing.List[str] = None, **kwargs
```

Comment:
It isn't clear to me the usage of each of these parameters from reading the function and current documentation, namely, `rows`, `keys` and `kwargs`. I would recommend adding more examples/adding clarification to the docstring. Suggestions below

Reply:
adding